packages/flutter/lib/src/painting/oval_border.dart

```diff
@@ -22,6 +25,25 @@ class OvalBorder extends CircleBorder {
   /// Create an oval border.
   const OvalBorder({ super.side, super.ovalness = 1.0});
 
+  @override
+  ShapeBorder scale(double t) => OvalBorder(side: side.scale(t), ovalness: ovalness);
```

Comment:
It is the same as circle border

Reply:
Yes but if you don't override them here and then run Oval border tests, it fails because of mismatched object type.